src/uiSelectController.js

```diff
@@ -557,12 +558,12 @@ uis.controller('uiSelectCtrl',
             return false;
           }
           var inputWidth = containerWidth - input.offsetLeft - 10;
-          if (inputWidth < 50) inputWidth = containerWidth;
+          if (inputWidth < minInputWidth) inputWidth = containerWidth;
           ctrl.searchInput.css('width', inputWidth+'px');
           return true;
         };
 
-    ctrl.searchInput.css('width', '10px');
+    ctrl.searchInput.css('width', minInputWidth+'px');
```

Comment:
Basically, this seemed like the simplest fix without re-working how this width is calculated, so that's why I went with it!